src/commands/other/waifu.ts

```diff
@@ -0,0 +1,58 @@
+import { BrandingColors } from '#lib/common/constants';
+import { LanguageKeys } from '#lib/i18n/LanguageKeys';
+import { EmbedBuilder } from '@discordjs/builders';
+import { parseURL } from '@sapphire/utilities';
+import { Command, InteractionArguments, RegisterCommand } from '@skyra/http-framework';
+import { applyLocalizedBuilder, createSelectMenuChoiceName, resolveKey } from '@skyra/http-framework-i18n';
+
+@RegisterCommand((builder) =>
+	applyLocalizedBuilder(builder, LanguageKeys.Commands.Waifu.RootName, LanguageKeys.Commands.Waifu.RootDescription).addStringOption((builder) =>
+		applyLocalizedBuilder(builder, LanguageKeys.Commands.Waifu.OptionRange) //
+			.addChoices(
+				createSelectMenuChoiceName(LanguageKeys.Commands.Waifu.KeyAll, { value: 'all' }),
+				createSelectMenuChoiceName(LanguageKeys.Commands.Waifu.KeyMediumQuality, { value: 'medium' }),
+				createSelectMenuChoiceName(LanguageKeys.Commands.Waifu.KeyHighQuality, { value: 'high' }),
+				createSelectMenuChoiceName(LanguageKeys.Commands.Waifu.KeyLowQuality, { value: 'low' })
+			)
+	)
+)
+export class UserCommand extends Command {
+	private readonly SIZES = Object.freeze({
+		medium: { min: 0, max: 50_000 },
+		high: { min: 50_000, max: 75_000 },
+		low: { min: 75_000, max: 100_000 },
+		all: { min: 0, max: 100_000 }
```

Comment:
Please restore the docs we had before, but move them here in each property, it'll be useful for tracking the information from the website.